addons/medical_treatment/ACE_Settings.hpp

```diff
@@ -1,11 +1,47 @@
 class ACE_Settings {
+    // Settings without EGVAR are new and don't need backwards compat
+    class GVAR(advancedBandages) {
+        displayName = CSTRING(advancedBandages_DisplayName);
+        description = CSTRING(advancedBandages_Description);
+        typeName = "BOOL";
+        value = 1;
+        // todo: I don't like that wound reopening requires this setting to be enabled, they should be independent
+    };
+    class GVAR(advancedDiagnose) {
+        displayName = CSTRING(advancedDiagnose_DisplayName);
+        description = CSTRING(advancedDiagnose_Description);
+        typeName = "BOOL";
+        value = 1;
+        // todo: this setting just disables some treatment options, remove?
+    };
+    class GVAR(advancedMedication) {
+        displayName = CSTRING(advancedMedication_DisplayName);
+        description = CSTRING(advancedMedication_Description);
+        typeName = "BOOL";
+        value = 1;
+    };
+    // todo: the strings for all three advanced bandages/medication/diagnose settings are terribly ambigious
     class EGVAR(medical,allowLitterCreation) {
```

Comment:
If we have to use EGVAR() for a setting, it either means that the setting is misplaced, or the functionality the setting is repsonsible for is misplaced.  I would suggest moving the allowLiterCreation and all other EGVAR() settings to the respective folder.

Since this is more a suggestion than an actual issue: PR approved.

Reply:
All the ones with `EGVAR` are that way for backwards compatibility as they will become `movedToSQF` once I convert everything to CBA settings

Reply:
If we want to throw it out the window we can change them all though

Reply:
Understood. I did not consider this.